source/assets/javascripts/common.js

```diff
@@ -3,12 +3,27 @@ import "popper.js";
 import "bootstrap";
 import "details-element-polyfill";
 import NoUiSlider from "nouislider";
+import "slick-carousel";
+import Rellax from "rellax";
```

Comment:
Are we using rellax? it's commented some lines below. Can we remove it?

Reply:
Actually not at the moment. It make sense delete it 👍